lib/experimental/html/class-wp-html-tag-processor.php

```diff
@@ -29,6 +32,144 @@
  * of patches to that input. Tokenizes HTML but does not fully
  * parse the input document.
  *
+ * ## Usage
+ *
+ * Use of this class requires three steps:
+ *
+ *  1. Create a new class instance with your input HTML document.
+ *  2. Find the tag(s) you are looking for.
+ *  3. Request changes to the attributes in those tag(s).
+ *
+ * Example
```

Comment:
```suggestion
 * Example:
```

Reply:
I'm going to merge these in, but I also question if they should be here. In my persistent quest to understand docblock rendering I think at one point I even had an issue with `Example:` that `Example` didn't have.